### PR TITLE
README.md redesigned for better understanding

### PR DESCRIPTION
**Pull request by sraghuvanshi**
Hi sir/ma'am, 
Greetings of the day!

While I was trying to run tvb-recon locally on my machine, I was finding it a little difficult to find all the required commands in the current documentation. I thought maybe other contributors are also facing the same problem. So I have redesigned the documentation and made it more readable and easily understandable. 

Please look into it.

P.S If you will say then I'll remove my name from the contributor's list as I've not contributed much yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,4 +361,14 @@ The automatized workflow is based on:
...
+
+
+### Contributors 
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not modify this section -->
+<!-- prettier-ignore -->
+| [<img src="https://avatars0.githubusercontent.com/u/674853?s=460&v=4" width="100px;"/><br /><sub><b>Lia Domide</b></sub>](https://github.com/liadomide)<br />        | [<img src="https://avatars2.githubusercontent.com/u/5189886?s=460&v=4" width="100px;"/><br /><sub><b>marmaduke woodman</b></sub>](https://github.com/maedoc/)<br /> | [<img src="https://avatars3.githubusercontent.com/u/3601158?s=400&v=4" width="100px;"/><br /><sub><b>Bogdan Neacsa</b></sub>](https://github.com/bogdanneacsa)<br />          | [<img src="https://avatars3.githubusercontent.com/u/1563810?s=400&v=4" width="100px;"/><br /><sub><b>Paula Sanz-Leon</b></sub>](https://github.com/pausz)<br /> | [<img src="https://avatars0.githubusercontent.com/u/2657768?s=400&v=4" width="100px;"/><br /><sub><b>Michael Burgstahler</b></sub>](https://github.com/twotribes)<br />    | [<img src="https://avatars3.githubusercontent.com/u/48951339?s=460&v=4" width="100px;"/><br /><sub><b>Saksham Raghuvanshi</b></sub>](https://github.com/sraghuvanshi)<br />                               |
```
Comment (liadomide) [inline README.md]: I would not keep this section here. 
Github repo already has the list of contributors nicely rendered.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,69 @@
...
 
 The mandatory inputs are DWI and T1 scans. Optionally, CT scans can be given as input, if sensors preprocessing is needed.
 
-We are using the [Pegasus WMS](https://pegasus.isi.edu/) in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
+We are using the <a href="https://pegasus.isi.edu/">Pegasus WMS</a> in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
+</p>
+
+<p align="center">
+  <img height="600" src="https://ars.els-cdn.com/content/image/1-s2.0-S1053811915002505-gr1.jpg">
+</p>
```
Comment (liadomide) [inline README.md]: This image should be clearly cited. I believe it is from article [2] in the Biobliography.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,69 @@
...
+    <img src="https://img.shields.io/badge/Pegasus%20WMS-4.9.3-%2349A13D"
+         alt="Gitter">
+  </a>
+  <a href="https://neurostars.org/">
+	  <img src="https://img.shields.io/badge/Chat-on%20Neurostars-%23673B90">
+</a>
+  <a href="https://saythanks.io/">
+      <img src="https://img.shields.io/badge/SayThanks.io-%E2%98%BC-1EAEDB.svg">
+  </a>
+  <a href="https://www.paypal.com/">
```
Comment (liadomide) [inline README.md]: It is not really fitting for our dev model to have this link here for donations

File: README.md [hunk 4]
```diff
@@ -1,31 +1,69 @@
...
+</p>
+<p>
+<hr>
+
+It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB or used independently for modeling.
 
 The mandatory inputs are DWI and T1 scans. Optionally, CT scans can be given as input, if sensors preprocessing is needed.
 
-We are using the [Pegasus WMS](https://pegasus.isi.edu/) in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
+We are using the <a href="https://pegasus.isi.edu/">Pegasus WMS</a> in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
```
Comment (liadomide) [inline README.md]: To be added:
`while our own code is GPL v3`

File: README.md (hunk 1 above)
Comment (sraghuvanshi) [reply README.md]: Sure, ma'am, I'll remove this section.

File: README.md (hunk 2 above)
Comment (sraghuvanshi) [reply README.md]: Yes, it is from the article [2] in the Bibliography. I thought it is representing the process clearly that's why I placed it here. Should I replace it? Also, ma'am, if you have any other image then please share it here.

File: README.md (hunk 3 above)
Comment (sraghuvanshi) [reply README.md]: Okay, ma'am, I'll replace it with TVB download link (https://www.thevirtualbrain.org/tvb/zwei/brainsimulator-software). Please let me know If you want any other link to be displayed here.

File: README.md (hunk 4 above)
Comment (sraghuvanshi) [reply README.md]: Sure, ma'am.

File: README.md (hunk 3 above)
Comment (sraghuvanshi) [reply README.md]: Also,ma'am, for the Say thanks button, you'll have to create an account in saythanks.io (https://saythanks.io/). Please let me know the link to your account, I'll link it to the button.

File: README.md (hunk 2 above)
Comment (liadomide) [reply README.md]: I also think it is a good image. I only suggested to have written close to the image where it is taken from

File: README.md (hunk 3 above)
Comment (liadomide) [reply README.md]: If I think better, I would replace the SayThanks button with a link towards our Jira installation https://req.thevirtualbrain.org/

File: README.md (hunk 2 above)
Comment (sraghuvanshi) [reply README.md]: Okay ma'am

File: README.md (hunk 3 above)
Comment (sraghuvanshi) [reply README.md]: Sure, ma'am. Any other button you would like to add?

Comment (liadomide) [reply README.md]: None comes to mind now. Thanks!

File: README.md (hunk 2 above)
Comment (liadomide) [reply README.md]: We tend to be less formal in our conversation. 
Thus, no need to address me with ma'am ;-)

Comment (sraghuvanshi) [reply README.md]: Okay sure :D

File: README.md (hunk 3 above)
Comment (sraghuvanshi) [reply README.md]: Okay, all the changes are done. Please let me know if any other change is required.

File: README.md [hunk 5]
```diff
@@ -1,31 +1,74 @@
...
+  <br>
+</h1>
+<h4 align="center">This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.</h4>
+<p align="center">
+  <a href="https://pegasus.isi.edu/downloads/">
+    <img src="https://img.shields.io/badge/Pegasus%20WMS-4.9.3-%2349A13D"
+         alt="Gitter">
+  </a>
+  <a href="https://neurostars.org/">
+	  <img src="https://img.shields.io/badge/Chat-on%20Neurostars-%23673B90">
```
Comment (popaula937) [inline README.md]: I would put the link towards tvb official forum: https://groups.google.com/forum/#!forum/tvb-users instead of neurostars

Comment (popaula937) [reply README.md]: The rest looks good from my side. Thank you for your contribution!

Comment (sraghuvanshi) [reply README.md]: Okay sure, sir.

File: README.md [hunk 6]
```diff
@@ -250,4 +366,4 @@ The automatized workflow is based on:
 [3] Andre Santos Ribeiro, Luis Miguel Lacerda, Hugo Alexandre Ferreira, Multimodal Imaging Brain Connectivity Analysis (MIBCA) toolbox, 2015 Jul 14. [Available here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4511822/)
 
 
-[![Build Status](https://travis-ci.org/the-virtual-brain/tvb-recon.svg?branch=master)](https://travis-ci.org/the-virtual-brain/tvb-recon) [![Coverage Status](https://coveralls.io/repos/github/the-virtual-brain/tvb-recon/badge.svg)](https://coveralls.io/github/the-virtual-brain/tvb-recon)
+[![Build Status](https://img.shields.io/badge/Build-error-critical)](https://travis-ci.org/the-virtual-brain/tvb-recon) [![Coverage Status](https://img.shields.io/badge/Coverage-unknown-informational)](https://coveralls.io/github/the-virtual-brain/tvb-recon)
```
Comment (popaula937) [inline README.md]: Forgot to comment on this change. Isn't this a link to a static badge? 
We need to show here the badge generated by travis for each build, so, I believe this link is correct: https://travis-ci.org/the-virtual-brain/tvb-recon.svg , but maybe we could remove: branch=master ?

Comment (sraghuvanshi) [reply README.md]: Yes, my bad, It should be a dynamic badge. I'll revert these changes. 
https://travis-ci.org/the-virtual-brain/tvb-recon.svg -  displays "build failing"
https://travis-ci.org/the-virtual-brain/tvb-recon.svg?branch=master - displays "build error"
In my opinion, I think we should use the link without branch=master.

**Pull request merged**